### PR TITLE
Add flag for working directory

### LINKED ISSUE #98: Add flag to set working directory for dunner
Currently dunner can be run from project root directory only and config file can be passed from anywhere using flag/present working directory by default.

Ref: https://github.com/leopardslab/dunner/blob/v2.1.3/pkg/docker/docker.go#L62

```
		hostMountFilepath          = "./"
```

This has pwd as default host mount file path. This can be made configurable.

In cases like CI systems/complex project structure, having a flag to set working directory can be useful. This can be done using a flag.

### PR DESCRIPTION
**Pull request by ayushjn20**
Resolves #98

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -55,6 +55,15 @@ func init() {
 		log.Fatal(err)
 	}
 
+	// Working directory
+	rootCmd.PersistentFlags().StringP("dir", "C", "./", "Working directory")
```
Comment (agentmilindu) [inline cmd/root.go]: Shall we change this to `context` as we already have something as `dir`

Comment (ayushjn20) [reply cmd/root.go]: Sounds good.

**Comment by agentmilindu** [comment]
I'm wondering why your PR contains a lot of other unrelated changes, like adding badges to README.md. etc.

**Comment by apoorvam** [comment]
Looks good. @ayushjn20 Please update changes in documentation too, if any.

**Comment by ayushjn20** [comment]
> Looks good. @ayushjn20 Please update changes in documentation too, if any.

Documentation must be in sync with `master` branch. We shall update the documentation just before another release to the `master` branch.

**Pull request merged**